.github/workflows/ci.yml

```diff
@@ -67,6 +76,19 @@ jobs:
     - name: Run pgx tests
       run: su postgres -c 'sh tools/build -pg${{ matrix.pgversion }} test-extension 2>&1'
 
+    - name: Build and install TimescaleDB
+      if: ${{ inputs.tsdb-commit != '' }}
+      run: |
+        apt-get install -y wget cmake gcc build-essential libkrb5-dev
```

Comment:
How about moving this into a standalone shell script?  `tools/install-timescaledb`?
We should already have build-essential and gcc.

I don't see where you're using wget.  If somehow curl is insufficient, we should switch our curl uses to wget so we don't need both.

We should just add cmake into the image.  It's useful and not too big (and we used to ship it anyway).

Does TimescaleDB really require Kerberos?  That's surprising.

Reply:
I added cmake and libkrb5-dev directly into the image. TimescaleDB really does require Kerberos, since it depends on libpq, which depends on Kerberos. I don't think TimescaleDB actually uses libpq's Kerberos support though.